src/components/Form.js

```diff
@@ -10,19 +10,26 @@ export default function Form() {
   const submitForm = (e) => {
     e.preventDefault();
     dispatch(addBookAsync({
-      title, author, item_id: Date.now(), category: 'action',
+      title, author, item_id: Date.now(), category: 'Action',
     }));
     setTitle('');
     setAuthor('');
   };
   return (
     <div>
-      <section className="mx-44 mt-11">
-        <h2>ADD NEW BOOK</h2>
-        <form action="" onSubmit={submitForm}>
-          <input type="text" className="border border-grey-900 h-10" value={author} onChange={(e) => setAuthor(e.target.value)} placeholder="Author" />
-          <input type="text" className="border border-grey-900 ml-10 h-10" value={title} onChange={(e) => setTitle(e.target.value)} placeholder="Title" />
-          <button type="submit" className="bg-blue-700 p-4 rounded-lg ml-10">ADD BOOK</button>
+      <section className="mt-11 pb-16">
+        <h2 className="font-montserrat font-bold text-gray-900 opacity-50 text-2xl pb-4">ADD NEW BOOK</h2>
+        <form action="" onSubmit={submitForm} className="">
+          <input type="text" className="border border-gray-300 h-14 pl-5 w-96" value={title} onChange={(e) => setTitle(e.target.value)} placeholder="Title" />
+          <input type="text" className="border border-gray-300 ml-10 w-96 h-14 pl-5" value={author} onChange={(e) => setAuthor(e.target.value)} placeholder="Author" />
+          <select name="cars" id="cars" className="h-14 w-72 ml-10 outline-none border border-gray-300 bg-white pl-5">
+            <option value="" disabled selected hidden>Category</option>
+            <option value="action">Action</option>
+            <option value="horror">Horror</option>
+            <option value="fiction">Fiction</option>
+            <option value="action">Action</option>
+          </select>
+          <button type="submit" className="ml-14 px-20 py-4 rounded bg-blue-400 font-robot font-bold text-white">ADD BOOK</button>
```

Comment:
Good Job working on the project requirements but It would be nice to follow design guideline and properly style this the book name font is overlapping the author section carefully look through the template and style your work like it is in the original design. Please follow design requirements. Please use the required fonts, Font color and font size 


This is your design:
![Screenshot 2023-08-03 at 5 07 21 PM](https://github.com/oluwatobi-fasasi/book-store/assets/79506105/61ee99ba-8844-40a7-a0da-2d8ec058d451)

Please follow this design:
![Screenshot 2023-08-03 at 5 06 47 PM](https://github.com/oluwatobi-fasasi/book-store/assets/79506105/e8a9a60e-4f52-4ccf-8630-284e903ea789)

Reply:
Thanks @DeliceLydia